app/assets/stylesheets/styles/pages.scss

```diff
@@ -184,10 +184,24 @@
     form {
       margin-top: 25px;
     }
+
+    .scroll-wrap {
+      position: relative;
+      top: 80px;
+
+      .scroll-item {
+        @extend %scroll-notification;
+        color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.